package/MDAnalysis/analysis/dssp/dssp.py

```diff
@@ -0,0 +1,293 @@
+In this example we will simply print a string representing 
+protein's secondary structure.
+
+.. code-block:: python
+    from MDAnalysis.tests.datafiles import PDB
+    from MDAnalysis.analysis.dssp import DSSP
+    u = mda.Universe(PDB)
+    s = ''.join(DSSP(u).run().results.dssp)
+    print(s)
+
+
+Calculating average secondary structure of a trajectory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Here we take a trajectory and calculate its average secondary structure,
+i.e. assign a secondary structure label 'X' to a residue if most of the frames
+in the trajectory got assigned 'X' label.
+
+.. code-block:: python
+    from MDAnalysis.analysis.dssp import DSSP, translate
+    from MDAnalysisTests.datafiles import TPR, XTC
+    u = mda.Universe(TPR, XTC)
+    long_run = DSSP(u).run(stop=20)
+    mean_secondary_structure = translate(
+        long_run.results.dssp_ndarray.mean(axis=0)
+        )
+    print(''.join(mean_secondary_structure)[:20])
+    # '-EEEEEE------HHHHHHH'
+
+Find parts of the protein that maintain their secondary structure during simulation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In this example, we will find residue groups that maintain their secondary structure
+along the simulation, and have some meaningful ('E' or 'H') secondary structure
+during more than set `threshold` share of frames. We will call these residues
+"persistent", for clarity, and label them according to the structure
+that they maintain during the run:
+
+.. code-block:: python
+    from MDAnalysis.analysis.dssp import DSSP, translate
+    from MDAnalysisTests.datafiles import TPR, XTC
+    u = mda.Universe(TPR, XTC)
+    threshold = 0.8
+
+    long_run = DSSP(u).run()
+    persistent_residues = translate(
+        long_run
+        .results
+        .dssp_ndarray
+        .mean(axis=0) > threshold
+    )
+    print(''.join(persistent_residues)[:20])
+    # '--EEEE----------HHHH'
+
+
+Functions
+---------
+
+.. autofunction:: get_hbond_map
+.. autofunction:: assign
+.. autofunction:: translate
+
+Analysis classes
+----------------
+
+.. autoclass:: DSSP
+   :members:
+   :inherited-members:
+
+   .. attribute:: results.dssp
+
+       Contains the time series of the DSSP assignment
+       as chars N×m :class:`numpy.ndarray` array with content
+       ``[frame, residue]``, where `structure_type` is one of three characters:
+       ['H', 'E', '-'], representing alpha-helix, sheet and loop, respectively.
+
+   .. attribute:: results.dssp_ndarray
+
+       Contains the one-hot encoding of the time series of the DSSP assignment
+       as chars N×mx3 :class:`numpy.ndarray` array with content
+       ``[frame, residue, encoding]``, where `encoding` is a (3,) shape
+       :class:`numpy.ndarray` of booleans with axes representing loop '-',
+       helix 'H' and sheet 'E', consequtively.
+"""
+
+import numpy as np
+
+from ..base import AnalysisBase
+from ... import Universe
```

Comment:
I *think* I would `from MDAnalysis import Universe` (or get it from MDAnalysis.core.universe)

Reply:
oh indeed, thanks -- just as you mentioned earlier in discussions